src/matchers/toBeHexadecimal/index.test.js

```diff
@@ -0,0 +1,31 @@
+import matcher from './';
+
+expect.extend(matcher);
+
+describe('.toBeHexadecimal', () => {
+  test('passes when given valid 6 digit hexadecimal', () => {
+    expect('#ECECEC').toBeHexadecimal();
+  });
+
+  test('passes when given valid 3 digit hexadecimal', () => {
+    expect('#000').toBeHexadecimal();
+  });
+
+  test('fails when given invalid hexadecimal', () => {
+    expect(() => expect('#eeggee').toBeHexadecimal()).toThrowErrorMatchingSnapshot();
+  });
+
+  test('fails when given non-string', () => {
```

Comment:
is it worth having a test for a string that isn't hexadecimal as well?

Reply:
Agreed. This boundary is already covered in the predicate test.